### ISSUE TITLE
warning with ruby 2.7

### ISSUE THREAD

**Issue by sylvek** [ruby-git/ruby-git#455]
### Subject of the issue
after upgraded my mac to ruby 2.7, i have warning when i use ruby-git lib.

```
/usr/local/lib/ruby/gems/2.7.0/gems/git-1.6.0/lib/git/lib.rb:1054: warning: Using the last argument as keyword parameters is deprecated
```
https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

### Your environment
* ruby 2.7.0p0
* git-1.6.0

**Comment by sylvek** [comment]
i don't have this warning with `git-1.5.0`

**Comment by dhs-rec** [comment]
Yeah, they're quite annoying, esp. since adding `-W:no-deprecated` to silence them doesn't work for me.
@sylvek, I guess you meant "Ruby", not "Python", right? Would you mind to change that?

**Comment by sylvek** [comment]
@dhs-rec thank you ☺️

**Comment by pyrmont** [comment]
This issue can be closed. An updated gem has not yet been pushed but this was fixed in #453.

**Comment by skcc321** [comment]
please push a new version, because that is so annoying. The only warning what I see all the time on any command.

**Comment by chris-roerig** [comment]
In the mean time, as in before the new gem is released, it is super easy to patch your installed version

https://github.com/ruby-git/ruby-git/pull/453/files

**Comment by jcouball** [comment]
Fixed in #453 and release in [v1.7.0](https://github.com/ruby-git/ruby-git/releases/tag/v1.7.0)
